CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ This release contains the [PDV refactor (#4111)](https://github.com/hasura/graph
 - server: add action-like URL templating for event triggers and remote schemas (fixes #2483)
 - server: change `created_at` column type from `timestamp` to `timestamptz` for scheduled triggers tables (fix #5722)
 - server: allow configuring timeouts for actions (fixes #4966)
+- server: accept only non-negative integers for batch size and refetch interval (close #5653) (#5759)
```

Comment:
I don't think we need a changelog atleast in the current form as "batch size" and "refetch interval" are pretty vague.